### ISSUE TITLE
[FEA]Support both list and non-list format for multi-hot feature input

### ISSUE THREAD

**Issue by jershi425** [NVIDIA/NVTabular#1022]
**Is your feature request related to a problem? Please describe.**
Currently, for multi-hot features, we have to first convert them into a list format and then process it with NVT. For example, the raw format of "genres" in the movielens data is like "drama|comedy". I have to first convert it to ["drama", "comedy"] using pandas and then applying label encoding using NVT. If the dataset is small, pandas is OK to do this job. But if the dataset is as big as Criteo dataset, this process becomes very inefficient. 

**Describe the solution you'd like**
I would be good if NVT can treat both list and strings as multi-hot input. Say if the input is string like "drama|comedy" or "drama,comedy", NVT can automatically treat them as multi-hot input.

**Comment by benfred** [comment]
you can do this with a lambadop in NVTabular -  For example to create a list column from strings like "drama|comedy" then categorical encode them in NVT would be something like:

```
genres = nvt.ColumnGroup(["genres"]) >> (lambda col: col.str.split("|")) >> nvt.ops.Categorify()
workflow = nvt.Workflow(genres)
workflow.fit_transform(dataset)
```
